test.js

```diff
@@ -20,6 +25,9 @@ const nonMatches = [
 	'#123hashtag',
 	'non/-repo#123',
 	'this/is/not/repo#123',
+	'thisorganisationnameistoolongxxxxxxxxxxx/foo#123',
+	'foo/thisrepositorynameistoolongxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx#123',
+	'foo/thisissuenumberistoolong#11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
```

Comment:
Can you add a code comment about why these are non-match. Like that the max length is this much.